server/src/utils/media.ts

```diff
@@ -377,8 +383,10 @@ export class ThumbnailConfig extends BaseConfig {
     return new ThumbnailConfig(config);
   }
 
-  getBaseInputOptions(): string[] {
-    return ['-skip_frame nointra', '-sws_flags accurate_rnd+full_chroma_int'];
+  getBaseInputOptions(videoStream: VideoStreamInfo, format?: VideoFormat): string[] {
+    return format?.formatName === 'mpegts'
```

Comment:
The issue is probably related to FFmpeg tickets [7950](https://trac.ffmpeg.org/ticket/7950) and [7895](https://trac.ffmpeg.org/ticket/7895). It'd be good to add a comment about why this condition is here.